crates/compiler/builtins/roc/Set.roc

```diff
@@ -43,9 +43,9 @@ isEq = \xs, ys ->
             else
                 Break Bool.false
 
-## An empty set.
-empty : Set k | k has Hash & Eq
-empty = @Set Dict.empty
```

Comment:
unrelated: why is there a bound on `k` here? for an empty set/dict that should not be needed